wp-rocket.php

```diff
@@ -29,7 +29,7 @@
 define( 'WP_ROCKET_WEB_VALID',             WP_ROCKET_WEB_MAIN . 'valid_key.php' );
 define( 'WP_ROCKET_WEB_INFO',              WP_ROCKET_WEB_MAIN . 'plugin_information.php' );
 define( 'WP_ROCKET_FILE',                  __FILE__ );
-define( 'WP_ROCKET_PATH',                  realpath( plugin_dir_path( WP_ROCKET_FILE ) ) . '/' );
+define( 'WP_ROCKET_PATH',                  realpath( dirname( WP_ROCKET_FILE ) ) . '/' );
```

Comment:
What is the idea behind changing this line?

Reply:
Probably because `plugin_dir_path` already comes with a trailing slash?

Reply:
@arunbasillal True. It does:

```php
function plugin_dir_path( $file ) {
	return trailingslashit( dirname( $file ) );
}
```

Reply:
I'd prefer this type of change though to be in a separate PR, as it's not part of replacing `cheating, uh` work.

What do you think @Tabrisrp?

Reply:
@ranss I reverted commit 5c43c3dda5d3eec5586196d8a8a9f62a7368f2cd. Why? The code changes are not related to `cheatin' uh` replacement. Instead, those types of changes should be submitted as a separate PR.